docs/general/administration/hardware-selection.md

```diff
@@ -0,0 +1,125 @@
+
+### System Memory (RAM)
+
+As a general rule of thumb, we recommend at least 8GB for Linux and 16GB for Windows, though significantly smaller amounts may work just fine.
+
+### Graphics Cards (GPUs)
+
+With modern systems, the media engines are usually located on the GPUs. Therefore, the hardware acceleration options available is entirely determined by the choice of GPUs.
+
+We recommend Intel graphics over Nvidia graphics because of the quality. We recommend avoiding AMD or Apple Silicon since they produce poor quality output. Encoder Quality: Intel > Nvidia > AMD > Apple
+
+The following is a list of commonly used codecs:
+
+- H.264 (AVC): Blu-Ray, most streaming services
+- H.265 (HEVC): Blu-Ray UHD
+- MPEG-2: DVD
+- VC-1: Blu-Ray
+- VP8, VP9, AV1: streaming services
+
+The following is a list of codecs Jellyfin supports transocding to:
+
+- H.264 (Most common transcode target)
+- H.265 (Limited supported by clients)
+
+For decoding support, the more codecs the better. However, there are only a few codecs that media is commonly available in, as listed above.
+
+#### Intel Graphics
+
+Please refer to [This Table](https://en.wikipedia.org/wiki/Intel_Quick_Sync_Video#Hardware_decoding_and_encoding) for supported codecs.
+
+Do Note that Intel ARC requres kernel version 6.2 or newer on Linux. Please check your distribution for details.
+
+#### Nvidia Graphics
+
+Please refer to [This Table](https://developer.nvidia.com/video-encode-and-decode-gpu-support-matrix-new) for supported codecs.
+
+#### AMD Graphics
+
+Please refer to [This Article](https://en.wikipedia.org/wiki/Video_Coding_Engine) for supported codecs.
+
+#### Apple Silicon
+
+Supported codecs are listed below:
+
+| Codec      | M1 | M1 Pro | M2 | M2 Pro |
+|------------|----|--------|----|--------|
+| H.264      | ✅  | ✅      | ✅  | ✅      |
+| H.265      | ✅  | ✅      | ✅  | ✅      |
+| ProRes     | ✅  | ✅      | ✅  | ✅      |
+| ProRes RAW | ❌  | 🔶      | 🔶  | 🔶      |
+| VP9        | ❌  | ❌      | 🔶  | 🔶      |
+| AV1        | ❌  | ❌      | ❌  | ❌      |
+
+Refer to the Pro column for Max and Ultra SOCs.
+
+✅ = Supported, 🔶 = Decode Only, ❌ = Not Supported.
+
+Please note that many hardware acceleration features are not available on MacOS for Jellyfin, and no Apple Silicon media engine drivers exist for other operating systems.
+
+### Storage
+
+There are 2 general types of storage that will be used on a Jellyfin server. Media libraries and Jellyfin files require different performance characteristics.
+
+#### Media Storage
+
+Anything with higher sequential access speed than bitrate of media will work.
+
+#### Jellyfin Files
+
+Jellyfin files will see lots of random access, therefore we recommend SSDs for Jellyfin files. AVOID mechanical drives that use SMR (Shingled Magnetic Recording) as they will result in a VERY poor experience. (But if you want your JF home page take 10 mins to load then sure.)
```

Comment:
```suggestion
Jellyfin files will see lots of random access, therefore we recommend SSDs for Jellyfin files. AVOID mechanical drives that use SMR (Shingled Magnetic Recording) as they will result in a VERY poor experience.
```

Kinda unnecessary snarky comment even though it it true lol

Reply:
Agreed. Official documentation pages need to keep an air of impartiality and professionalism. The project is represented by these pages.